lib/src/Pos/Util/Servant.hs

```diff
@@ -64,12 +66,23 @@ module Pos.Util.Servant
     , inRouteServer
 
     , applyLoggingToHandler
+    , ValidJSON
+    , Tags
+    , mapRouter
+    , WalletResponse(..)
```

Comment:
A lot of code was moved from `wallet-new` into `Pos.Util.Servant` as the "location" of servant-related stuff that is useful to other packages.